From 418ebe8df46d1b7b36a15a2d5a5ad01c31baa073 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Mon, 9 Oct 2023 14:26:43 +0200
Subject: [PATCH] com.webos.service.sleep.api.json.in: Add API for
 suspend/activityStart

Fixes:

2023-10-09T12:01:20.064805Z [1544.043133747] kern.warning sleepd [] <default-lib> LS_REQUIRES_SECURITY {"SERVICE":"com.webos.service.power","CATEGORY":"/suspend","METHOD":"activityStart"} Service security groups don't allow method call.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 files/sysbus/com.webos.service.sleep.api.json.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/files/sysbus/com.webos.service.sleep.api.json.in b/files/sysbus/com.webos.service.sleep.api.json.in
index 8d2b79e..bc5f3c4 100644
--- a/files/sysbus/com.webos.service.sleep.api.json.in
+++ b/files/sysbus/com.webos.service.sleep.api.json.in
@@ -2,6 +2,7 @@
 "sleep.operation": [
 "com.palm.sleep/com/palm/power/activityEnd",
 "com.palm.sleep/com/palm/power/activityStart",
+"com.webos.service.power/suspend/activityStart",
 "com.palm.sleep/com/palm/power/clientCancelByName",
 "com.palm.sleep/com/palm/power/forceSuspend",
 "com.palm.sleep/com/palm/power/identify",
@@ -21,6 +22,7 @@
 "sleep.management": [
 "com.palm.sleep/com/palm/power/activityEnd",
 "com.palm.sleep/com/palm/power/activityStart",
+"com.webos.service.power/suspend/activityStart",
 "com.palm.sleep/com/palm/power/clientCancelByName",
 "com.palm.sleep/com/palm/power/forceSuspend",
 "com.palm.sleep/com/palm/power/identify",
